setup.py

```diff
@@ -3,23 +3,38 @@
 from setuptools import setup, find_packages
 
 
-with open('README.rst') as f:
+with open('README.md') as f:
     readme = f.read()
 
 with open('LICENSE') as f:
     license = f.read()
 
+install_requires = ['setuptools==36.2.7',
+                    'hvac==0.2.17+git.b817da4',
+                    'requests==2.18.4',
+                    'python_daemon==2.1.2',
+                    ]
+dependency_links = ['https://github.com/mracter/hvac/archive/b0811f0446760125baefea6383711035104e6df1.zip'
+                    + '#egg=hvac-0.2.17+git.b817da4']
```

Comment:
Prefer not using `+` for these multi-line strings... see further down.

Also, I don't actually know how to install things from git in `setup.py`... 😕

Reply:
I've put installation instructions on `README.md`.